src/shared/components/CountdownTimer/components/TimerList/TimerList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { TimerItem } from './components/TimerItem';
+import { Periods } from '../../../../api/constants/periods';
+import './TimerList.scss';
+
+interface TimerListProps {
+  values: number[];
+}
+
+export const TimerList = ({ values }: TimerListProps) => {
+  const periods = [
+    Periods.DAYS,
+    Periods.HOURS,
+    Periods.MINUTES,
+    Periods.SECONDS,
+  ];
```

Comment:
move to constants folder

Reply:
moved to shared/api/constants/periodsArray.ts